io/io-q&a.md

```diff
@@ -201,7 +204,8 @@
 
 ### 42. Ce se întâmplă dacă facem open de mai multe ori consecutiv pe același fișier?
 
-   Se va adăuga un nou file descriptor în FDT și se va crea un alt open file structure pentru fd-ul creat.
+   La fiecare apel de `open()`, se va adăuga un nou file descriptor în FDT și se va crea un alt open file structure pentru fd-ul creat.
```

Comment:
I'd rather not have these changes applied now. The information would be categorized as "Nice to know", and we'll have this section added as a whole later on. Don't worry I'll take this info and use it when the time comes.

Reply:
Sure thing. An extras section sounds great!

Reply:
Ok, then, let's keep this one.